data/remnant/remnant 1 introduction.txt

```diff
@@ -3820,7 +3820,7 @@ mission "Remnant: Logistics 1"
 			`Eon leads you to the bustling cafeteria, chattering about ship design since the Alpha Wars. He orders a couple of hot drinks and leads you to a quiet table overlooking the docks. You try your drink. It's strongly flavored but a bit odd, like someone's memory of what coffee should taste like, reconstituted from local foods.`
 			`	Eon leaves his untouched as he launches into conversation. "Those of us in logistics have noticed that you are spending time in our systems, and we wanted to see if you would be available to pick up work for us. It would be delivery work like the generator you brought for the housing project."`
 			choice
-				`	(Nod)`
+				`	(Nod.)`
```

Comment:
This feels really redundant and unnecessary. It might be grammatically correct, but it still feels useless. It's not really a sentence, or a statement, just an action that's even delineated as not being a real part of the dialog or text by the parentheses.

Reply:
Yeah, every other action choice so far has punctuation. I'm happy to change all of that if that becomes the agreed-upon style (and I will mention it in the style guide), but so far this is how it has been in Endless Sky.

Reply:
Action choices should still be grammatically correct. A one word sentence is still a sentence and should therefore end in a period.